media-gfx/gscan2pdf/gscan2pdf-2.1.7.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils perl-module
```

Comment:
Is eutils somehow being used in this?

Reply:
Yes for optfeature.

Reply:
Right, it was a legit question because I wasn't sure :)